### DISCUSSION TITLE
Send form data from React Native application

### DISCUSSION THREAD

**Discussion by DZ8540** [adonisjs/core#3667] [Help] [answered]
Hello there, i have a very ugly problem. My employee decide this problem already three days, and i don't know how decide this problem!

He sended form data test query to my api, from react native application via axios, with file (on image below, we send it without file).

<img width="633" alt="image" src="https://user-images.githubusercontent.com/50152205/162162824-635dcf66-f3c9-4013-8026-888b47fa6e10.png">

It will sended data like this.

<img width="216" alt="image" src="https://user-images.githubusercontent.com/50152205/162163408-2c1793a1-defd-4457-9999-f781e1773476.png">

But my api sended response like this. My api not see his fields(

<img width="325" alt="image" src="https://user-images.githubusercontent.com/50152205/162163818-b6f31367-eeda-4106-bd17-9d30dacb822b.png">

My validator.

<img width="597" alt="image" src="https://user-images.githubusercontent.com/50152205/162164253-b8f371d0-c48c-4176-a7b3-43823c44156c.png">

How i should take his data?

## UPD

He sended data via fetch and this is work, but we need use axios(

**Comment by thetutlage** [comment, marked as answer]
Seems like an issue with manually setting the `content-type`. The content type should have the multipart boundary. Also, in the axios docs, they do not set the content-type manually. https://github.com/axios/axios#manual-formdata-passing

**Reply by DZ8540** [reply]
Yes! He installed axios@0.21.4 and it is work! Thank you very much, how always best)

Axios issue: https://github.com/axios/axios/issues/4577
